tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/TestHelper.cs

```diff
@@ -88,6 +90,15 @@ public async Task<Process> StartDotnetTestSample(MockTracerAgent agent, string a
                 throw new Exception($"application not found: {sampleAppPath}");
             }
 
+            // Ensure we have the CI Environment variables in the target process
```

Comment:
Hmmm... This seems like a good way to not be able to repro CI issues locally, no? 😅 We remove them all by default intentionally... are you _sure_ you don't want to just set these manually per-test to be safe?

Reply:
moved to optional in https://github.com/DataDog/dd-trace-dotnet/pull/5269/commits/c400bda63c9512ec05e6a0a2551dbe6c8e92c123